src/components/project.js

```diff
@@ -5,11 +5,9 @@ import Img from "gatsby-image"
 import useBookmarkState from "./bookmark_state"
 import BookmarkIcon from "./bookmark_icon"
 
-import isDOMavailable from "../isDOMavailable.js"
-
 import "./project.css"
 
-const Project = ({ project }) => {
+const Project = ({ isInitialRoute, project }) => {
```

Comment:
We have to pass this from the page because it turns out all of the projects render three times and for only the first is `isInitialRoute()` true (since momentarily after the first render, the route history is updated, then the next two renders happen (presumably because of image fetching or something like that).